db/knex_migrations/2024-08-24-000-add-cache-bust.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+    return knex.schema
+      .alterTable("monitor", function (table) {
+          table.boolean("cache_bust").notNullable().defaultTo(false);
+      });
```

Comment:
fixed the linting error properly

```suggestion
        .alterTable("monitor", function (table) {
            table.boolean("cache_bust").notNullable().defaultTo(false);
        });
```